### PR TITLE
New Cheat Sheet: Aircrack-ng

### PR DESCRIPTION
**Pull request by vduddu**
## Description of new Instant Answer, or changes

Commands for aircrack-ng

## People to notify

@vduddu

---

Instant Answer Page: https://duck.co/ia/view/aircrackng_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@vduddu Thanks for the pull request! Unfortunately, we're only accepting contributions and improvements to a limited portion of the search space.
            
The staff and community are currently focused on making DuckDuckGo the [best search engine for programmers](https://forum.duckduckhack.com/t/duckduckhack-programming-mission-overview/53). To ensure we're making as much progress as possible, **we are only accepting Instant Answers and updates related to the programming mission**.
    
If this is not related to the Programming Mission, it will likely be closed but you can:
    
1. Help us prevent further confusion by telling us where you found out about DuckDuckHack or where we could've alerted you earlier.
2. Keep this code handy! We'll be expanding into other search topics in the future!
3. Join us at our new home: https://forum.duckduckhack.com/

If this is related to the Programming Mission, it will be reviewed by staff and the community as soon as possible!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by daxtheduck** [comment]
@vduddu Thanks for the pull request! Unfortunately, we're only accepting contributions and improvements to a limited portion of the search space.
            
The staff and community are currently focused on making DuckDuckGo the [best search engine for programmers](https://forum.duckduckhack.com/t/duckduckhack-programming-mission-overview/53). To ensure we're making as much progress as possible, **we are only accepting Instant Answers and updates related to the programming mission**.
    
If this is not related to the Programming Mission, it will likely be closed but you can:
    
1. Help us prevent further confusion by telling us where you found out about DuckDuckHack or where we could've alerted you earlier.
2. Keep this code handy! We'll be expanding into other search topics in the future!
3. Join us at our new home: https://forum.duckduckhack.com/

If this is related to the Programming Mission, it will be reviewed by staff and the community as soon as possible!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by jophab** [comment]
@vduddu You have 2 files in this PR. Remove the file of john the ripper

File: share/goodie/cheat_sheets/json/aircrackng.json [hunk 1]
```diff
@@ -0,0 +1,458 @@
+{
+    "id": "aircrackng_cheat_sheet",
+    "name": "Aircrackng",
+    "description": "Cheatsheet and commands for aircrack-ng suite",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/aircrackng.json]: "Commands for aircrack-ng suite"

File: share/goodie/cheat_sheets/json/aircrackng.json [hunk 2]
```diff
@@ -0,0 +1,458 @@
...
+                "key": "--essid <ESSID>",
+                "val": "specify a single ESSID (short -e)"
+            },
+            {
+                "key": "--essids <FILE>",
+                "val": "read a list of ESSIDs out of that file (short -E)"
+            },
+            {
+                "key": "--clients <file>",
+                "val": "read a list of MACs out of that file (short -D)"
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/aircrackng.json]: Start the above val fields with capital letter

File: share/goodie/cheat_sheets/json/aircrackng.json [hunk 3]
```diff
@@ -0,0 +1,458 @@
...
+                "key": "--deauth <count>",
+                "val": "Deauthenticate 1 or all stations (-0)"
+            },
+            {
+                "key": "--fakeauth <delay>",
+                "val": "Fake authentication with AP (-1)"
+            },
+            {
+                "key": "--interactive",
+                "val": "interactive frame selection (-2)"
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/aircrackng.json]: Start the above val field with capital letter. Check the whole file for the same

**Comment by pjhampton** [comment]
Closing this due to inactivity. 


If you are looking for something more substantial to work through that will have a big impact for the users of DuckDuckGo, please check out the [DuckDuckHack Issues Table](https://duckduckhack.com/issues). 🚀  😄

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=?Tgva6uOjLv)

**Pull request closed without merging**